jquery.unobtrusive-ajax.js

```diff
@@ -91,7 +91,7 @@
         $.extend(options, {
             type: element.getAttribute("data-ajax-method") || undefined,
             url: element.getAttribute("data-ajax-url") || undefined,
-            cache: !!element.getAttribute("data-ajax-cache"),
+            cache: (element.getAttribute("data-ajax-cache") || "").toLowerCase() === "true",
```

Comment:
This doesn't work when the value is the Boolean value `true`. Whatever we do here we might as well have it work with all of `['true', true, 'false', false]`.